examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -22,12 +26,16 @@ architect(
         "tsconfig.app.json",
         "tsconfig.json",
         "@npm//mime",
+        "@npm//zone.js",
+        "@npm//tslib",
+        "@npm//typescript",
         "@npm//@angular/cli",
         "@npm//@angular/core",
         "@npm//@angular/router",
         "@npm//@angular/platform-browser-dynamic",
         "@npm//@angular-devkit/architect-cli",
         "@npm//@angular-devkit/build-angular",
+        "//projects/frontend-lib:build"
```

Comment:
ooh, this is the dream, the missing dependency edge! `ng serve` is sufficient workflow to get updates to libs

Reply:
Still facing an issue with the linker. The one we discussed yesterday. And the ci failure which I am unable to reproduce locally. 

>  ng serve is sufficient workflow to get updates to libs
This depends if we will follow the Angular CLI or Nx model. Angular CLI uses pre-compiled libraries. NX uses ts path mappings. Both of them have pros and cons.